documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,233 @@
+  capacity:
+    storage: 100Gi
+  *claimRef:*
+    *apiVersion: v1*
+    *kind: PersistentVolumeClaim*
+    *name: data-0-my-cluster-kafka-2*
+    *namespace: myproject*
+    *resourceVersion: "39113"*
+    *uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea*
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: failure-domain.beta.kubernetes.io/zone
+          operator: In
+          values:
+          - eu-west-1c
+        - key: failure-domain.beta.kubernetes.io/region
+          operator: In
+          values:
+          - eu-west-1
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: gp2-retain
+  volumeMode: Filesystem
+----
++
+In the example, the following properties are deleted:
++
+[source,shell,subs="+quotes,attributes"]
+----
+claimRef:
+  apiVersion: v1
+  kind: PersistentVolumeClaim
+  name: data-0-my-cluster-kafka-2
+  namespace: myproject
+  resourceVersion: "39113"
+  uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea
+----
+
+. Deploy the Cluster Operator.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl apply -f install/cluster-operator -n _my-project_
+----
+
+. Recreate your cluster.
++
+Follow the steps depending on whether or not you have all the `KafkaTopic` resources needed to recreate your cluster.
++
+--
+*_Option 1_*: If you have *all* the `KafkaTopic` resources that existed before you lost your cluster, including internal topics such as committed offsets from `__consumer_offsets`:
+
+. Recreate all `KafkaTopic` resources.
++
+It is essential that you recreate the resources before deploying the cluster, or the Topic Operator will delete the topics.
+
+. Deploy the Kafka cluster.
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl apply -f _kafka.yaml_
+----
+--
++
+--
+*_Option 2_*: If you do not have all the `KafkaTopic` resources that existed before you lost your cluster:
+
+. Deploy the Kafka cluster, as with the first option, but without the Topic Operator by removing `topicOperator: {}` from the Kafka resource before deploying.
++
+If you include the Topic Operator in the deployment, the Topic Operator will delete all the topics.
+
+. Run an `exec` command to one of the Kafka broker pods to open the ZooKeeper shell script.
++
+For example, where _my-cluster-kafka-0_ is the name of the broker pod:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl exec _my-cluster-kafka-0_ bin/zookeeper-shell.sh localhost:2181
+----
+. Delete the whole `/strimzi` path to remove the Topic Operator storage:
++
+[source,shell,subs="+quotes,attributes"]
+----
+deleteall /strimzi
+----
+. Enable the Topic Operator by redeploying the Kafka cluster with `topicOperator: {}` to recreate the `KafkaTopic` resources.
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  #...
+  entityOperator:
+    *topicOperator: {}*
```

Comment:
and of course here we could use a comment to specify the user topic operator configuration instead of curly braces and maybe mentioning/showing `{ }` just for using the default one.

Reply:
I've added a callout:

> <1> Here we show the default configuration, which has no additional properties.
> You specify the required configuration using the properties described in xref:type-EntityTopicOperatorSpec-reference[].